skll/learner.py

```diff
@@ -1386,7 +1386,7 @@ def train(self, examples, param_grid=None, grid_search_folds=3,
         if grid_search:
             if not grid_objective:
                 raise ValueError("You must specify a grid objective "
-                                 "if doing grid search.")
+                                 "or turn off grid search.")
```

Comment:
IMHO this type of error message should contain two pieces: 1) a statement of what went wrong, followed by 2) the suggested action for fixing it...something like "grid_search set to True but no grid objective provided. You must either specify a grid objective or turn off grid search." I know I'm picking nits here.

Reply:
Sure, that's a point well taken. I was trying to be succinct but I think perhaps being verbose is the better option here.